index.html

```diff
@@ -172,56 +172,65 @@ <h1>Automatically updating img tag's `sizes` attribute without sacrificing perfo
             alt="img-grid-img"
             test-path="+%3C%3E%5B%5D%7B%7D%7C%5C%5E%25.jpg"
             test-params="{}"
-            sizes="auto"
+            ix-sizes="auto"
           />
           <img
             data-test-id="sizes-grid"
+            ix-sizes="auto"
             alt="img-grid-img"
             test-path="+%3C%3E%5B%5D%7B%7D%7C%5C%5E%25.jpg"
             test-params="{}"
-            sizes="auto"
           />
           <img
             data-test-id="sizes-grid"
+            ix-sizes="auto"
             alt="img-grid-img"
             test-path="+%3C%3E%5B%5D%7B%7D%7C%5C%5E%25.jpg"
             test-params="{}"
-            sizes="auto"
           />
           <img
             data-test-id="sizes-grid"
+            ix-sizes="auto"
             alt="img-grid-img"
             test-path="+%3C%3E%5B%5D%7B%7D%7C%5C%5E%25.jpg"
             test-params="{}"
-            sizes="auto"
           />
+          <section>
+            <img
+              data-test-id="sizes-grid"
+              alt="img-grid-img"
+              ix-sizes="auto"
+              test-path="+%3C%3E%5B%5D%7B%7D%7C%5C%5E%25.jpg"
+              test-params="{}"
+            />
+        </section>
```

Comment:
Added another nested image here just to ensure we were correctly accounting for other type of container elements.